service/routes/rum.js

```diff
@@ -0,0 +1,125 @@
+/* Endpoints for Real User Monitoring of detect and perf results */
+
+'use strict';
+
+const polyfillio = require('../../lib');
+const metrics = require('../metrics');
+const express = require('express');
+
+let mysql;
+if (process.env.MYSQL_DSN) {
+	require('promise-mysql').createConnection(process.env.MYSQL_DSN).then(conn => {
+		mysql = conn;
+	});
+}
```

Comment:
I think just throwing in all situations is probably best.  If you don't want a mySQL connectionn you can just not set the DSN